dags/dag_content/nextflow_tower_functions.py

```diff
@@ -2,19 +2,20 @@
 from sagetasks.nextflowtower.utils import TowerUtils
 
 
-def create_and_open_tower_workspace(platform: str, workspace_id: str) -> TowerUtils:
+def create_and_open_tower_workspace(tower_access_token : str, platform: str, workspace_id: str) -> TowerUtils:
     """Creates and opens Nextflow Tower Workspace using sagetasks TowerUtils class
 
     Args:
-        platform (str): Which Nextflow Tower instance to run the workflow on. 
+        tower_access_token (str): Which Nextflow Tower Access Token to use for workflow run
+        platform (str): Which Nextflow Tower instance to run the workflow on.
         workspace_id (str): The ID number of the workspace you wish to run the workflow in
-    
+
     Returns:
         tower_utils (TowerUtils): Authenticated Tower Utils class
-    
+
     """
     client_args = TowerUtils.bundle_client_args(
-        auth_token=Variable.get("TOWER_ACCESS_TOKEN"), platform=platform
+        tower_access_token="TOWER_ACCESS_TOKEN", auth_token=Variable.get(tower_access_token), platform=platform
```

Comment:
Why are you providing this value to `bundle_client_args()`? While that function allows additional values to be provided thanks to `**kwargs`, it doesn't use this value. It seems like you only need this value for `Variable.get()`. 

Also, you can set the default value to `"TOWER_ACCESS_TOKEN"` so you don't have to change all of the DAGs. Since it would have a default value, this argument should be moved to the end of `create_and_open_tower_workspace()`.